pkg/buildconfig/constants.go

```diff
@@ -0,0 +1,10 @@
+// Code generated by pkg/codegen/config/main.go. DO NOT EDIT.
+// Package buildconfig contains a set of exported constants that represent configuration variables of Rancher at build-time.
+package buildconfig
+
+const (
+	CspAdapterMinVersion = "103.0.0+up3.0.0"
```

Comment:
At the risk of stating the obvious here, we will want to be careful not to accidentally downgrade anything when fixing conflicts to merge this PR.
For instance, Fleet `0.9.0` has been un-RCed.

Reply:
Yup, this is normal. I keep fixing conflicts while the PR is open, which is OK. We plan to merge it shortly after 2.8.0 is out. We won't need to migrate all other configurable values to the new system immediately. We can do so gradually - one value at a time.